pyfolio/utils.py

```diff
@@ -381,190 +374,69 @@ def estimate_intraday(returns, positions, transactions, EOD_hour=23):
     return corrected_positions
 
 
-def to_utc(df):
-    """
-    For use in tests; applied UTC timestamp to DataFrame.
+def clip_returns_to_benchmark(rets, benchmark_rets):
     """
+    If the returns is longer than the benchmark's, limit strategy returns.
```

Comment:
This description is a bit ambiguous. I would naively interpret "longer than" to mean "has more observations than", but it looks to me like the actual behavior here is that we remove any values from `rets` that are older than the oldest observation in `benchmark_rets`. I also wouldn't know what "limit" means without more context.

I would describe the currently-implemented behavior here as something like:

```python
def clip_returns_to_benchmark(rets, benchmark_rets):
    """Drop entries from rets that precede the first entry in benchmark_rets.
    """
```

That said, is there a reason we're only clipping the left side here? Naively, I'd expect to have similar problems if rets/benchmark_rets don't align in either direction.

Reply:
Interesting, I had just moved the code without thinking about it. The function now clips from both ends.